src/krux/krux_settings.py

```diff
@@ -198,6 +198,8 @@ class CNCSettings(SettingsNamespace):
     depth_per_pass = NumberSetting(float, "depth_per_pass", 0.03125, [0.0001, 10000])
     part_size = NumberSetting(float, "part_size", 3.5, [0.0001, 10000])
     border_padding = NumberSetting(float, "border_padding", 0.0625, [0.0001, 10000])
+    head_type = CategorySetting("head_type", "router", ["router", "laser"])
```

Comment:
I would love to see the string "router" and "laser" as CONSTS, as we did with `CNC_FILE_DRIVER`. My suggestion would be `CNC_HEAD_ROUTER` and `CNC_HEAD_LASER`